poseidon-permutation/src/permutation.rs

```diff
@@ -134,8 +136,8 @@ impl<'a, F: PrimeField> Instance<'a, F> {
     /// This implementation is based on the unoptimized Sage implementation
     /// `poseidonperm_x5_254_3.sage` provided in Appendix B of the Poseidon paper.
     fn unoptimized_permute(&mut self) {
-        let R_f = self.parameters.rounds.full() / 2;
-        let R_P = self.parameters.rounds.partial();
+        let R_f = self.parameters.rounds.r_F / 2;
+        let R_P = self.parameters.rounds.r_P;
```

Comment:
we can add these getters so that the API doesn't change too much

Reply:
the same for `elements`

Reply:
There are already such methods :P And someone told me recently that it's better to use full names ;)

Reply:
I still claim it is - just that it was inherited with these fugly shortcuts.